### DISCUSSION TITLE
Scrollbar ancora visibili

### DISCUSSION THREAD

**Discussion by IceManXXXA** [lifeisfoo/react-guida-pratica#10] [Q&A] [answered]
Salve, 
dopo aver seguito le istruzioni del capitolo 4 le scrollbar sono ancora visibili, nonostante abbia aggiunto il file style.css con le istruzioni riportate nel libro:
*::-webkit-scrollbar {
  display: none;
}

Qualcuno di voi ha avuto lo stesso problema?

uso firefox 116.0.1 (64bit), e inoltre la stringa 'import "./style.css";' è riportata correttamente nel file index.js

grazie mille!

**Comment by lifeisfoo** [comment]
Puoi verificare se il file css è stato caricato dal browser aprendo l'inspector (tasto destro -> analizza) e cercando l'elemento `<style>` corrispondente all'interno di `<head>`.

![image](https://github.com/lifeisfoo/react-guida-pratica/assets/3725610/885ec505-ed78-4982-ad17-d365ff2d4f53)

Puoi dirmi che sistema operativo stai usando e incollare qui uno screenshot di quello che vedi nel browser?

Grazie

**Comment by IceManXXXA** [comment]
Salve,

questo è quello che vedo, mi sembra corretto:
![React](https://github.com/lifeisfoo/react-guida-pratica/assets/141451629/bbddc55f-60e3-4022-8faf-be568fe01ecb)

utilizzo win10:

![React01](https://github.com/lifeisfoo/react-guida-pratica/assets/141451629/5a9fe07d-42f2-4092-a389-ccf18694b54d)


Grazie per la celerità!

**Comment by lifeisfoo** [comment, marked as answer]
Lo stile delle scrollbar non è definito da nessuno standard, quindi ogni browser fa un po' come vuole.
Tutto il codice del libro è stato provato con Chrome/Chromium (webkit).

Per nascondere tutte le scrollbar su Firefox è necessario aggiungere questa regole al file `style.css`.

```
* {
  scrollbar-width: none
}
```

Per lasciare la barra leggermente visibile si potrebbe usare `thin` al posto di `none` ma non è la stessa cosa.

A questi due link trovi tutta la storia per il supporto allo stile delle scrollbar in Firefox e, nel secondo, la proposta di adeguamente a un nuovo standard per renderlo uniforme tra i browser.

https://bugzilla.mozilla.org/show_bug.cgi?id=77790
https://bugzilla.mozilla.org/show_bug.cgi?id=1745850

**Reply by IceManXXXA** [reply]
Perfetto, con questa aggiunta funziona grazie mille per il supporto!

complimenti per il libro, è ben strutturato, l'ho trovato molto utile

Buone cose ;)
